wp-content/themes/storefront-child/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "storefront-child",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "pickee",
+  "license": "MIT",
+  "private": false,
+  "devDependencies": {
+    "breakpoint-sass": "^2.7.1",
+    "sass": "^1.30.0",
+    "susy": "2.2.14"
+  },
+  "scripts": {
+    "compile:css": "sass sass/sass/style.scss style.css --style=compressed --watch"
```

Comment:
@beamjet We can use 2 scripts and make it more clear:
```
"compile:css": "sass sass/sass/style.scss style.css --style=compressed",
"compile:css:watch": "yarn compile:css --watch"
```
And then update the Procfile to use the `watch` one.